client/my-sites/plans/jetpack-plans/product-lightbox/index.tsx

```diff
@@ -1,12 +1,163 @@
+import { Button } from '@automattic/components';
+import { useCallback, useState } from 'react';
+import Modal from 'react-modal';
 import { SelectorProduct } from '../types';
+import { Icons } from './icons/product-icons';
+import { Tags } from './icons/tags';
 import './style.scss';
 
 type Props = {
 	product: SelectorProduct;
+	isVisible: boolean;
+	onClose: () => void;
 };
 
-const ProductLightbox: React.FC< Props > = ( { product } ) => {
-	return <div className="product-lightbox__main-wrapper">{ product.displayName }</div>;
+type Option = {
+	value: string;
+	label: string;
+	checked?: boolean;
+};
+type OptionProps = {
+	options: Option[];
+	onChange: ( value: string ) => void;
+};
+
+const Options: React.FC< OptionProps > = ( { options, onChange } ) => {
+	return (
+		<>
+			{ options.map( ( option ) => {
+				return (
+					<label key={ option.value } className="product-lightbox__variants-option-label">
+						<input
+							type="radio"
+							name="variant-select[]"
+							value={ option.value }
+							checked={ option.checked }
+							onChange={ ( el ) => {
+								onChange( el.target.value );
+							} }
+						/>
+						<span>{ option.label }</span>
+					</label>
+				);
+			} ) }
+		</>
+	);
+};
+
+const ProductLightbox: React.FC< Props > = ( { product, isVisible, onClose } ) => {
+	const close = useCallback( () => onClose?.(), [ onClose ] );
+
+	const [ checked, setChecked ] = useState( '1' );
+
+	return (
+		<Modal
+			className="product-lightbox__modal"
+			overlayClassName="product-lightbox__modal-overlay"
+			isOpen={ isVisible }
+			onRequestClose={ close }
+			htmlOpenClassName="ReactModal__Html--open"
+		>
+			<div className="product-lightbox__content-wrapper">
+				<div className="product-lightbox__detail">
+					<div className="product-lightbox__detail-header">
+						{ Icons.backup }
+						<h2>{ product.displayName }</h2>
+					</div>
+					<div className="product-lightbox__detail-desc">
+						<p>
+							Protect your site or store. Save every change with real-time cloud backups, and
+							restore in one click from anywhere.
+						</p>
```

Comment:
I assume the strings will be i18n ready when the PR is ready for review, right?

Reply:
No, I actually thought it would be added on later stages when splitting this all to particular components because some texts will be taken from the 'product' object I assume. And this PR is simply to add styling.